src/components/InstructionsModal/InstructionsModal.tsx

```diff
@@ -45,7 +45,10 @@ export const InstructionsModal = ({
         >
           {title}
         </h2>
-        <p style={{ padding: '0 24px 24px', whiteSpace: 'pre-wrap' }}>{children}</p>
+        <p
+          style={{ padding: '0 24px 24px', whiteSpace: 'pre-wrap' }}
+          dangerouslySetInnerHTML={{ __html: children }}
```

Comment:
if we're considering using `dangerouslySetInnerHTML` on children which i think comes from user input - should we consider pulling in https://github.com/cure53/DOMPurify to sanitize it a little?

Reply:
It's actually already safe because of backend sanitization!
https://github.com/HumanSignal/label-studio/blob/eece1c8d39aa3850829bbc9b9b74833b2c5fba26/label_studio/projects/serializers.py#L88-L91